aiohttp/web_fileresponse.py

```diff
@@ -37,6 +38,14 @@
 
 NOSENDFILE: Final[bool] = bool(os.environ.get("AIOHTTP_NOSENDFILE"))
 
+# File extensions for IANA encodings that will be checked in the order defined.
```

Comment:
Since we are offering a guarantee that they are checked in order, we should validate that in a test as well.

Reply:
The tests already do it, otherwise they wouldn't pick up the `hello.txt.br` at all